src/middleware/common.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import * as cors from 'cors';
+import * as parser from 'body-parser';
```

Comment:
```suggestion
import { urlencoded, json } from 'body-parser';
```
then you can use bellow: 
```ts
router.use(urlencoded({ extended: true }));
router.use(json());
```